packages/template-retail-react-app/app/hooks/use-product.js

```diff
@@ -15,7 +15,7 @@ const OUT_OF_STOCK = 'OUT_OF_STOCK'
 const UNFULFILLABLE = 'UNFULFILLABLE'
 
 // TODO: This needs to be refactored.
-export const useProduct = (product) => {
+export const useProductData = (product) => {
```

Comment:
I see that the name of this hook conflicts with our commerce-sdk-react one. That's why we wanted to change it. What's a clearer name for it? 🤔 

Since we're deriving things from the given `product` data, how about `useDerivedProduct`?